README.md

```diff
@@ -7,14 +7,14 @@ Confluent's Golang Client for Apache Kafka<sup>TM</sup>
 
 Features:
 
-- **High performance** - confluent-kafka-go is a lightweight wrapper around
-[librdkafka](https://github.com/edenhill/librdkafka), a finely tuned C
+- **High performance** - `confluent-kafka-go` is a lightweight wrapper around
```

Comment:
I don't like this style, both confluent-kafka-go and librdkafka are proper product names to some degree, while the fixed-width style to me indicates some command or syntax thing.
If you want these names high-lighted I'd suggest bold.

Reply:
That sounds fine. In Markdown, the `` ` `` syntax typically denotes [`code`](https://www.markdownguide.org/basic-syntax/#code) such as an executable name or command flag. So I often see other repos displaying their program names that way in their readme's/docs. I do find it a little more readable with the program name called out that way, because when actually rendered the distracting backticks are replaced with a convenient grey background. But it's a trivial thing, and happy to go either way.